src/v/pandaproxy/rest/test/consumer_group.cc

```diff
@@ -30,6 +30,7 @@ namespace {
 
 auto get_consumer_offsets(
   http::client& client,
+  net::unresolved_address& host,
```

Comment:
Can this commit be dropped if `listener_idx` is used?

Reply:
Probably, once I add `listener_idx` I'll try out the original version to the unit tests.

Reply:
I'm removing the tests from this PR to reduce review friction. They'll come separately.